wagtail/wagtailadmin/utils.py

```diff
@@ -171,9 +171,11 @@ def send_mail(subject, message, recipient_list, from_email=None, **kwargs):
     return django_send_mail(subject, message, from_email, recipient_list, **kwargs)
 
 
-def send_notification(page_revision_id, notification, excluded_user_id):
+def send_notification(page_revision_id, notification, notification_user_id):
     # Get revision
     revision = PageRevision.objects.get(id=page_revision_id)
+    # User
+    User = get_user_model()
```

Comment:
Why not use directly `get_user_model()` directly where it’s useful, during context definition?